authentication/handler.go

```diff
@@ -176,7 +168,7 @@ func jwtErrorToProblem(err error) error {
 
 	if strings.HasPrefix(err.Error(), "parse with claims failed:") ||
 		strings.HasPrefix(err.Error(), "failed to validate claims:") {
-		return custom_problems.InvalidToken(old_errors.Cause(err).Error())
+		return custom_problems.InvalidToken(errors.Unwrap(err).Error())
```

Comment:
Is this handled in the same way in v5? I think the `.Cause` call was needed for some internal reason. Guessing we don't have tests here?

Reply:
I guess it was just due to the old `github.com/SKF/v2/jwt` returning `pkg/errors` errors, which was changed in the latest version...